### PR TITLE
fix: Unexpected token ...

### PR DESCRIPTION
**Pull request by flohhhh**
**Observation:**
fuse.js is not transpiled in es5

In production in some cases (with `webviews`
which have the version Chrome 57) we have
`Uncaught SyntaxError : Unexpected token ...`
because the Spread syntax (...) is supported
 from version 60.

Specific import allows you to have the
 in es5 library.

Issue in fuse.js repo:
https://github.com/krisk/Fuse/issues/553

**other solutions considered:** :
- Add rule for webpack :
   - poses a problem on other projects that have not ejected webpack config like cozy store app
- Fork fuse.js

### CODE REVIEW

File: src/ducks/categories/CategoryChoice.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import sortBy from 'lodash/sortBy'
-import Fuse from 'fuse.js'
+import Fuse from 'fuse.js/dist/fuse.js'
```
Comment (Crash--) [inline src/ducks/categories/CategoryChoice.jsx:4]: you should import the minified version: 

```
➜  fuse.js git:(master) ✗ ls -la dist
total 744
-rw-r--r--   1 quentinvalmori  staff  41793 10 jui 14:59 fuse.basic.common.js
-rw-r--r--   1 quentinvalmori  staff  28864 10 jui 14:59 fuse.basic.esm.js
-rw-r--r--   1 quentinvalmori  staff  10013 10 jui 14:59 fuse.basic.esm.min.js
-rw-r--r--   1 quentinvalmori  staff  44470 10 jui 14:59 fuse.basic.js
-rw-r--r--   1 quentinvalmori  staff  14607 10 jui 14:59 fuse.basic.min.js
-rw-r--r--   1 quentinvalmori  staff  62512 10 jui 14:59 fuse.common.js
-rw-r--r--   1 quentinvalmori  staff   6894 10 jui 14:59 fuse.d.ts
-rw-r--r--   1 quentinvalmori  staff  41031 10 jui 14:59 fuse.esm.js
-rw-r--r--   1 quentinvalmori  staff  15439 10 jui 14:59 fuse.esm.min.js
-rw-r--r--   1 quentinvalmori  staff  66491 10 jui 14:59 fuse.js
-rw-r--r--   1 quentinvalmori  staff  22871 10 jui 14:59 fuse.min.js
```

Comment (flohhhh) [reply src/ducks/categories/CategoryChoice.jsx:4]: Why ? it will be modified in the build.

Comment (Crash--) [reply src/ducks/categories/CategoryChoice.jsx:4]: A link to the fuse's issue can be useful to know when to remove this import

**Review by Crash-- — APPROVED** [review]

**Comment by ptbrowne** [comment]
https://github.com/Megan-TA/webpack-bundle-syntax-check-plugin#readme pourrait être intéressant pour checker le fait que le bundle soit parsable en es5 (le readme est en chinois mais le code lui est bien lisible :)).

**Pull request merged**